src/ParseServer.js

```diff
@@ -143,6 +143,11 @@ class ParseServer {
     enableSingleSchemaCache = false,
     __indexBuildCompletionCallbackForTests = () => {},
   }) {
+    // verify parse-server is running on node >= 4.6
+    if (process.versions.node < '4.6') {
```

Comment:
we have a problem here...  @TylerBrock ! https://twitter.com/tylerbrock/status/879807420838576132